yellow-paper/src/preprocess/InstructionSet/InstructionSet.js

```diff
@@ -149,7 +149,7 @@ const INSTRUCTION_SET_RAW = [
         "Category": "Compute - Comparators",
         "Flags": [
             {"name": "indirect", "description": INDIRECT_FLAG_DESCRIPTION},
-            {"name": "inTag", "description": IN_TAG_DESCRIPTION_NO_FIELD},
```

Comment:
is this constant still hanging around?

Reply:
Yes we still need it for the bitwise ops.